### PR TITLE
add test for debounce by specific fields

### PR DESCRIPTION
**Pull request by guysegal**

### CODE REVIEW

File: src/main/java/stream/StreamConfiguration.java [hunk 1]
```diff
@@ -19,6 +19,7 @@ public class StreamConfiguration extends Properties {
     public StreamConfiguration() {
         super();
         put(StreamsConfig.APPLICATION_ID_CONFIG, Config.GROUP_ID);
+        put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 1000);
```
Comment (sarialalem1) [inline src/main/java/stream/StreamConfiguration.java]: I suggest we make this configurable, we might see some cases where we need this to be higher or less. especially this implementation being a long running aggregation

File: tests/specs/tests.ts__
Comment (sarialalem1) [inline tests/specs/tests.ts__:1]: Delete this file

**Review by sarialalem1 — APPROVED** [review]

File: src/main/java/stream/StreamConfiguration.java (hunk 1 above)
Comment (guysegal) [reply src/main/java/stream/StreamConfiguration.java]: done

File: tests/specs/tests.ts__
Comment (guysegal) [reply tests/specs/tests.ts__:1]: done

**Pull request merged**